src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -135,7 +135,10 @@ public virtual void ConfigureArgumentParser(OptionSet optionSet, ChocolateyConfi
                      option => configuration.ListCommand.IdStartsWith = option != null)
                  .Add("detail|detailed",
                      "Detailed - Alias for verbose.",
-                     option => configuration.Verbose = option != null);
+                     option => configuration.Verbose = option != null)
+                 .Add("ignore-pinned",
+                     "Ignore Pinned - Ignore pinned packages. Defaults to false.",
+                     option => configuration.ListCommand.IgnorePinned = option != null);

```

Comment:
If going with my other comment, this could maybe be changed to:

```suggestion
                     option => configuration.ListCommand.ListPinnedPackages = option is null);

```